packages/ifpack/src/Ifpack_SerialTriDiSolver.h

```diff
@@ -127,7 +127,7 @@ Examples using Ifpack_SerialTriDiSolver can be found in the Epetra test director
 */
 
 //=========================================================================
-class EPETRA_LIB_DLL_EXPORT Ifpack_SerialTriDiSolver :
```

Comment:
Will removing this export not break some customer's usage of this class from the DLL?  Why is this export being removed?  Why not leave this export?

Reply:
As described in the commit message, the export indicates epetra, but this class is in ifpack - as a result this was never exported in the first place and the only effect it has is breaking the build of ifpack on windows. It is certainly possible that this should have an ifpack export, but I felt this to be the more conservative change.

Reply:
Okay, I see that in commit c4d2092687991bfc657b0440ab78d2652a39bec4.  I should have looked at that.

With that, exports are only being added in this PR so I don't see any reason not to merge this PR.